cli/tests/test_global_opts.rs

```diff
@@ -48,10 +48,13 @@ fn test_version() {
 
     let stdout = test_env.jj_cmd_success(test_env.env_root(), &["--version"]);
     let sanitized = stdout.replace(|c: char| c.is_ascii_hexdigit(), "?");
+    let expected = [
+        "jj ?.??.?\n",
+        "jj ?.??.?-????????????????????????????????????????\n",
+    ];
     assert!(
-        sanitized == "jj ?.??.?\n"
-            || sanitized == "jj ?.??.?-????????????????????????????????????????\n",
-        "{sanitized}"
+        expected.contains(&sanitized.as_str()),
+        "`jj version` output: {stdout:?}.\nSanitized: {sanitized:?}\nExpected one of: {expected:?}"
```

Comment:
I would probably have used literal newline characters instead of escapes, but up to you

Reply:
I keep forgetting that rewrapping library. reindent? Maybe later.